app/models/dialog/terraform_template_service_dialog.rb

```diff
@@ -0,0 +1,92 @@
+require "JSON"
```

Comment:
I think you mean `require "json"`
```
>> require 'JSON'
<internal:/usr/lib/ruby/vendor_ruby/rubygems/core_ext/kernel_require.rb>:86:in `require': cannot load such file -- JSON (LoadError)
```

Also for delay load purposes (keeping memory usage low) we prefer to put the require in the method/class that it is used in.  In this case you can add it right above here https://github.com/ManageIQ/manageiq-providers-embedded_terraform/pull/75/files#diff-153c36f8373e1ea8b9918d4a35cdd505014f31e78f92c7702ffd53eb32892e27R26